chips/stm32f303xc/src/adc.rs

```diff
@@ -0,0 +1,740 @@
+        JQOVF_SLV OFFSET(26) NUMBITS(1) [],
+        // Analog watchdog 3 flag of the slave ADC
+        AWD3_SLV OFFSET(25) NUMBITS(1) [],
+        // Analog watchdog 2 flag of the slave ADC
+        AWD2_SLV OFFSET(24) NUMBITS(1) [],
+        // Analog watchdog 1 flag of the slave ADC
+        AWD1_SLV OFFSET(23) NUMBITS(1) [],
+        // End of injected sequence flag of the slave ADC
+        JEOS_SLV OFFSET(22) NUMBITS(1) [],
+        // End of injected conversion flag of the slave ADC
+        JEOC_SLV OFFSET(21) NUMBITS(1) [],
+        // Overrun flag of the slave ADC
+        OVR_SLV OFFSET(20) NUMBITS(1) [],
+        // End of regular sequence flag of the slave ADC
+        EOS_SLV OFFSET(19) NUMBITS(1) [],
+        // End of regular conversion of the slave ADC
+        EOC_SLV OFFSET(18) NUMBITS(1) [],
+        // End of Sampling phase flag of the slave ADC
+        EOSMP_SLV OFFSET(17) NUMBITS(1) [],
+        // Slave ADC ready
+        ADRDY_SLV OFFSET(16) NUMBITS(1) [],
+        // Injected Context Queue Overflow flag of the master ADC
+        JQOVF_MST OFFSET(10) NUMBITS(1) [],
+        // Analog watchdog 3 flag of the master ADC
+        AWD3_MST OFFSET(9) NUMBITS(1) [],
+        // Analog watchdog 2 flag of the master ADC
+        AWD2_MST OFFSET(8) NUMBITS(1) [],
+        // Analog watchdog 1 flag of the master ADC
+        AWD1_MST OFFSET(7) NUMBITS(1) [],
+        // End of injected sequence flag of the master ADC
+        JEOS_MST OFFSET(6) NUMBITS(1) [],
+        // End of injected conversion flag of the master ADC
+        JEOC_MST OFFSET(5) NUMBITS(1) [],
+        // Overrun flag of the master ADC
+        OVR_MST OFFSET(4) NUMBITS(1) [],
+        // End of regular sequence flag of the master ADC
+        EOS_MST OFFSET(3) NUMBITS(1) [],
+        // End of regular conversion of the master ADC
+        EOC_MST OFFSET(2) NUMBITS(1) [],
+        // End of Sampling phase flag of the master ADC
+        EOSMP_MST OFFSET(1) NUMBITS(1) [],
+        // Master ADC ready
+        ADRDY_MST OFFSET(0) NUMBITS(1) []
+    ],
+    // Common control register
+    CCR [
+        // VBAT enable
+        VBATEN OFFSET(24) NUMBITS(1) [],
+        // Temperature sensor enable
+        TSEN OFFSET(23) NUMBITS(1) [],
+        // VREFINT enable
+        VREFEN OFFSET(22) NUMBITS(1) [],
+        // ADC clock mode
+        CKMODE OFFSET(16) NUMBITS(2) [],
+        // Direct memory access mode for dual ADC mode
+        MDMA OFFSET(14) NUMBITS(2) [],
+        // DMA configuration (for dual ADC mode)
+        DMACFG OFFSET(13) NUMBITS(1) [],
+        // Delay between 2 sampling phases
+        DELAY OFFSET(8) NUMBITS(4) [],
+        // Dual ADC mode selection
+        DUAL OFFSET(0) NUMBITS(5) []
+    ],
+    // Common regular data register for dual mode
+    CDR [
+        // Regular data of the slave ADC
+        RDATA_SLV OFFSET(16) NUMBITS(16) [],
+        // Regular data of the master ADC
+        RDATA_MST OFFSET(0) NUMBITS(16) []
```

Comment:
These descriptive comments are awesome, but it would be great if you could make them doc comments so they show up in the docs (i.e. replace `//` --> `///`)